docs/BacwardCompatibility.md

```diff
@@ -0,0 +1,224 @@
+
+We will update the OpenAPI version values, but they cannot be used for API communications.
+
+## Default Taxonomy
+Fybrik widely uses [Taxonomy](https://fybrik.io/v1.1/concepts/taxonomy/) to defines the terms and related values that 
+need to be commonly understood and supported across the components in the system. Taxonomy provides a mechanism for all 
+Fybrik components to interact using a common dialect. Taxonomy defines a set of immutable structural JSON schemas, or 
+"taxonomies" for resources deployed in Fybrik. However, since the taxonomy is meant to be configurable, a taxonomy.json 
+file is referenced from these schemas for any definition that is customizable. The taxonomy.json file is generated from 
+a base taxonomy and zero or more taxonomy layers:
+- The base taxonomy is maintained by the project and includes all the structural definitions that are subject to 
+customization (e.g.: tags, actions).
+- The taxonomy layers are maintained by users and external systems that add customizations over the base taxonomy 
+(e.g., defining specific tags, actions), therefore the layer are beyond Fybrik responsibility. 
+
+The structural JSON schemas are autogenerated files in the 
+[taxonomy](https://github.com/fybrik/fybrik/tree/master/charts/fybrik/files/taxonomy) direction.
+- The `fybrik_application.json` and `fybrik_module.json` files are generated from corresponded CRDs, so their 
+compatability is covered by the CRDs compatibility.
+- The `datacatalog.json`, `infraatributes.json` and `policymanager.json` files are generated form files in the 
+[pkg/model](https://github.com/fybrik/fybrik/tree/master/pkg/model), therefore any changes in the directory should be  
+validated for backward compatibility.
+- The basic [taxonomy file](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/taxonomy/taxonomy.json), is
+autogenerated from the both above sources. However, users are able to overwrite it, as describe by [Using a Custom 
+Taxonomy for Resource Validation](https://fybrik.io/v1.1/tasks/custom-taxonomy/). 
+
+Unfortunately all these files do not have version definition and it can be complicated to separate files from different 
+releases. In order to be able to compare the files, I suggest to add an entry `version` and modify the 
+[json-schema-generator](https://github.com/fybrik/json-schema-generator) project to generate the entry. It's an optional 
+entry, so we don't have to change the go structures, but will be able to validate the content of the 
+`fybrik-taxonomy-config` configuration map. Due to possibility of user modifications of the basic taxonomy file I suggest 
+in addition to the version, add the data of the file generation and maybe its CRC. 
+
+*Note:* Should we create a taxonomy json file for the Credential Management connector?
+*Note:* Should we provide a [default metric taxonomy](https://fybrik.io/v1.1/tasks/infrastructure/#add-a-new-attribute-definition-to-the-taxonomy)
```

Comment:
They are defined here https://fybrik.io/v1.1/tasks/infrastructure/#how-to-define-infrastructure-attributes

Reply:
i'm not sure that we are talking about the same.